models/classes/runner/map/QtiRunnerMap.php

```diff
@@ -508,4 +508,23 @@ private function getOverriddenOptionsRepository(): OverriddenOptionsRepositoryIn
         /** @noinspection PhpIncompatibleReturnTypeInspection */
         return $this->getServiceLocator()->get(OverriddenOptionsRepositoryInterface::SERVICE_ID);
     }
+
+    /**
+     * Tool/option activation status in frontend is checked by fuzzy matched categories. This step was added
+     * in order to have the same behavior for tool/option deactivation through overrides
+     */
+    private function removeFuzzyMatchedCategories(array $categoriesMap, string $categoryId): array
```

Comment:
maybe filterCategories, or something else? not sure that fuzzy matched is defined definition

Reply:
Fuzzy matched, because it's matched disregarding case/prefix/whitespace/dashes/underscores. The used term is the same as in FE. `filterCategories` name is not specific enough in my opinion. Do you have other ideas?